### PR TITLE
Task 03: Word stats

### PR DESCRIPTION
**Pull request by RakValery**

### CODE REVIEW

File: tasks/task03/ Word_stats.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import re
+print("Welcome to Word Statistic Tool!")
```
Comment (minotru) [inline tasks/task03/ Word_stats.py]: in this task you do not need to read the strings from the console.
assume you have a list of strings defined in the top like:
```python
texts = [
   "Hello, World!",
   "The world is mine",
   "Hello, how are you?"
]
```

Comment (RakValery) [reply tasks/task03/ Word_stats.py]: Ok
will fix it

File: tasks/task03/ Word_stats.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import re
+texts = ["Hello, World!", "The world is mine", "Hello, how are you?"]
+dictw = {}
+i = 0
+for istr in texts:
+    for iword in re.split(r'[:; ,*.!?\n]+',istr.lower()):
+        if iword == "":
+            continue
+        if dictw.get(iword, "e404") == "e404":
```
Comment (minotru) [inline tasks/task03/ Word_stats.py]: In Python, use None to denote absence of a value. dict.get(key) will return None by default if such key is not present. So you can write:
if dict.get(key) is None:

In your case you actually do not need the value, you only check if the key is present or not.
In this case, use 

if key not in dict:

File: tasks/task03/ Word_stats.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+            dictw[iword] = [1, i]
+        else:
+            dictw[iword][0] += 1
+    i += 1
+print("word\t\tcount\tfirst line\n----------------------------------")
+for iword in dictw.keys():
```
Comment (minotru) [inline tasks/task03/ Word_stats.py]: Nice trick here:
There is dict.iteritems() which returns key-value pairs:
for word, stats in dictw.iteritems():


Moreover, your value is a list, it supports unpacking, so you can write:
for word, (count, line) in dictw.iteritems()

File: tasks/task03/ Word_stats.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import re
+texts = ["Hello, World!", "The world is mine", "Hello, how are you?"]
+dictw = {}
+i = 0
+for istr in texts:
```
Comment (minotru) [inline tasks/task03/ Word_stats.py]: There is a function enumerate(collection) which yields an iterable of pairs (index, item), it is very useful when you need both item and its index, just like here:

for i, istr in enumerate (texts):

**Review by minotru — APPROVED** [review]

**Review by minotru — APPROVED** [review]

**Pull request merged**